custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -31,6 +31,7 @@ def validate_phone_number(string_value):
     if string_value is None or string_value.strip() in ['', '0']:
         return ''
     else:
+        assert int(string_value) > 0
         phone_number = str(int(string_value))
```

Comment:
removing the leading zero is actually required for putting it into international format (for example if it's an 11-digit number with a leading zero)